File: testdata/dart/messages/analyse.dart
void demo(dynamic err, String value, String message) {
  throw Exception("exception fail");
  throw StateError('state fail');
  throw ArgumentError('argument fail');
  throw FlutterError("""flutter fail""");
  throw FormatException('format fail');

  print("print fail");
  debugPrint('debug fail');

  throw Exception("hello $value");
  throw StateError(prefix + 'ignored');
  throw err;
  print(err);
  debugPrint(message);
}

File: testdata/go/messages/analyse.go
package sample

import (
	"errors"
	"fmt"
	"log"
)

func demo(err error, value string) {
	errors.New("new fail")
	fmt.Errorf(`format fail`)
	log.Print("print fail")
	log.Printf("printf fail: %s", value)
	log.Println("println fail")
	panic("panic fail")

	errors.New(prefix + "ignored")
	fmt.Errorf(err.Error())
	log.Print(err)
	panic(err)
}
